__tests__/cmds/categories/index.test.ts

```diff
@@ -13,10 +14,19 @@ describe('rdme categories', () => {
 
   afterEach(() => nock.cleanAll());
 
-  it('should error if no api key provided', () => {
-    return expect(categories.run({})).rejects.toStrictEqual(
+  it('should prompt for login if no API key provided', async () => {
```

Comment:
Want to add tests for when the user fully fills out the login form to get an API key? Might be a lot to test/mock out. I'm happy with this as-is.

Reply:
This `loginFlow` function has complete test coverage via the `login` command tests so I'm not gonna worry about that for now.